api/migrations/0002_data_migration.py

```diff
@@ -108,6 +109,26 @@ def _write_languages_and_levels(self):
         )
         LanguageAndLevel.objects.bulk_create(language_and_level_objects)
 
+    def _write_non_teaching_help(self):
+        HelpType = self.apps.get_model(APP_NAME, "NonTeachingHelp")
+
+        help_types = []
+        for id_and_name in (
+            ("cv_write_edit", "CV and cover letter (write or edit)"),
+            ("cv_proofread", "CV and cover letter (proofread)"),
+            ("mock_interview", "Mock interview"),
+            ("job_search", "Job search"),
+            ("career_strategy", "Career strategy"),
+            ("linkedin", "LinkedIn profile"),
+            ("career_switch", "Career switch"),
+            ("portfolio", "Portfolio for creative industries"),
+            ("uni_abroad", "Entering a university abroad"),
+            ("translate_docs", "Translation of documents"),
+        ):
+            help_types.append(HelpType(id=id_and_name[0], name=id_and_name[1]))
```

Comment:
It's better to use unpacking:

`for id, name in (...):`

Reply:
Done